### PR TITLE
perf(PLAT-1604): improve pagespeed performance

### PR DESCRIPTION
**Pull request by bcamarneiro**
## Why?

The performance value from PageSpeed metrics were a bit on the lower side (as per the spreadsheet in the ticket)

## How?

- Removed GTM and imported plausible directly
- Converted all images from png/jpg to webp
- Resized some of the biggest images
- The big image in the landing page hero now has sources (1 for desktop and a smaller one for mobile)
- Lazy loading youtube videos - these will only load when clicked but still show a placeholder while not clicked
- Reimplemented landing page hero animations with css/js to avoid having to load framer motion;
- Removed a couple of dependencies that were not being used in the codebase (posthog is being loaded through a script so I think it doesn't make sense to have it in package.json)
- Lazy load "globe" video in the loading page (it will load async once the user gets near the element - "near" as in 50px from it)
- I think that's all

## Tickets?

- [PLAT-1604 1](https://linear.app/fleekxyz/issue/PLAT-1604/address-each-of-the-mobile-page-speed-issues-on-the-seo-technical)

## Results
(on the left the updated version and on the right current version)
image

## Contribution checklist?

- [ ] The commit messages are detailed
- [ ] The `build` command runs locally
- [ ] Assets or static content are linked and stored in the project
- [ ] Document filename is named after the slug
- [ ] You've reviewed spelling using a grammar checker
- [ ] For documentation, guides or references, you've tested the commands and steps
- [ ] You've done enough research before writing

## Security checklist?

- [ ] Sensitive data has been identified and is being protected properly
- [ ] Injection has been prevented (parameterized queries, no eval or system calls)
- [ ] The Components are escaping output (to prevent XSS)

## References?

Optionally, provide references such as links

## Preview?

demo_url: https://acoustic-battery-puny.on-fleek.app/

### CODE REVIEW

**Review by victorgaard — COMMENTED** [review]
Amazing improvement on the mobile performance! 👏👏  I'll review more thoroughly, but just wanted to double check if you are importing the correct `IBM Plex Sans` font. For ref, check the letter `a` in the paragraph.

Fleek.xyz:
image


Demo link:
image

**Comment by bcamarneiro** [comment]
@victorgaard thanks! I'll double check

**Review by heldrida — APPROVED** [review]

File: src/styles/typography.css [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+    unicode-range: U+0000-00FF, U+0131, U+0152-0153, U+02BB-02BC, U+02C6, U+02DA,
+      U+02DC, U+0304, U+0308, U+0329, U+2000-206F, U+20AC, U+2122, U+2191,
+      U+2193, U+2212, U+2215, U+FEFF, U+FFFD;
+  }
+  @font-face {
+    font-family: 'IBM Plex Sans';
+    font-style: normal;
+    font-weight: 500;
+    font-display: swap;
+    src: url('/fonts/ibm-plex-sans/zYX9KVElMYYaJe8bpLHnCwDKjSL9AIFsdP3pBms.woff2')
```
Comment (victorgaard) [inline src/styles/typography.css:107]: probably just a typo from file name but this url is returning a 404 (see `Platform`)

image

File: src/styles/typography.css [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+    unicode-range: U+0000-00FF, U+0131, U+0152-0153, U+02BB-02BC, U+02C6, U+02DA,
+      U+02DC, U+0304, U+0308, U+0329, U+2000-206F, U+20AC, U+2122, U+2191,
+      U+2193, U+2212, U+2215, U+FEFF, U+FFFD;
+  }
+  @font-face {
+    font-family: 'IBM Plex Sans';
+    font-style: normal;
+    font-weight: 700;
+    font-display: swap;
+    src: url('/fonts/ibm-plex-sans/zYX9KVElMYYaJe8bpLHnCwDKjWr7AIFsdP3pBms.woff2')
```
Comment (victorgaard) [inline src/styles/typography.css:118]: ^

File: .gitignore [hunk 3]
```diff
@@ -23,8 +23,14 @@ pnpm-debug.log*
...
 # search
 meili_data
 
 # vscode settings folder
-.vscode/
\ No newline at end of file
+.vscode/
+
+#fleek
+fleek.config.json
```
Comment (victorgaard) [inline .gitignore:36]: will this be needed for the website?

File: src/styles/typography.css (hunk 2 above)
Comment (bcamarneiro) [reply src/styles/typography.css]: this should be fixed

File: src/styles/typography.css (hunk 1 above)
Comment (bcamarneiro) [reply src/styles/typography.css]: this should be fixed

File: .gitignore (hunk 3 above)
Comment (victorgaard) [reply .gitignore:36]: oh ok that makes sense :) question was more out of curiosity

**Pull request merged**